app/controllers/ingredients_controller.rb

```diff
@@ -2,4 +2,25 @@ class IngredientsController < ApplicationController
   def index
     @ingredients = Ingredient.all
   end
+
+  def new
+    @ingredient = Ingredient.new
+  end
+
+  def create
+    @ingredient = Ingredient.new(ingredient_params)
+    respond_to do |format|
+      if @ingredient.save
+        format.html { redirect_to ingredients_path }
+      else
+        format.html { render action: 'new' }
+      end
+    end
```

Comment:
We don't support any other format than `html` so we can skip `respond_to` block.